contracts/GuestList.vy

```diff
@@ -55,13 +90,31 @@ def set_bouncer(new_bouncer: address):
     self.bouncer = new_bouncer
 
 
+@view
+@internal
+def _yfi_in_makerdao(user: address) -> uint256:
+    proxy: address = self.proxy_registry.proxies(user)
```

Comment:
does the registry return 0x0 if the user doesnt have 1? thinking if thats the case we can break right away and return 0